app/controllers/api/v1/courses_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
         paginate json: @courses
       end
 
-      def show
+      def show_by_recruitment_cycle
         @courses = Course.all.includes(:sites, :provider, :site_statuses)
```

Comment:
This will just render all courses... dont we need to check the `params` the year and then do something like

`@courses = Course.where(recruitment_cycle: params[:year])`

Reply:
@dankmitchell I thought we didn't have this info in the db yet, so until we sort that out (which is a bigger piece of work), we'd just dump it all.